sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/EndpointConfig.java

```diff
@@ -38,8 +38,7 @@ public TestGlobalProperty initGlobalValues() {
     property.setEsPort(System.getenv("sunbird_es_port"));
     property.setIndexType(System.getenv("sunbird_es_index_type"));
     property.setIndex(System.getenv("sunbird_es_index"));
-    // System.getenv("sunbird_default_channel")
-    property.setSunbirdDefaultChannel("sunbird-dev");
+    property.setSunbirdDefaultChannel(System.getenv("sunbird_default_channel"));
```

Comment:
sunbird_default_channel variable need to be added inside env file of this project.